.buildkite/pipeline.yml

```diff
@@ -31,9 +31,10 @@ steps:
   # Lint
   #################
   - label: "🧹 Lint"
+    key: "lint"
     command: .buildkite/lint.sh
     plugins:
-      - automattic/bash-cache#add/cocoapods-support: ~
+      - automattic/bash-cache#v1.1.0: ~
```

Comment:
version mismatch with other steps here – because you're not using the anchor to DRY it 😜 . See comment below for suggestions (+ Anchor crash course Slack thread).